proto/viam/app/slammapping/v1/slammapping.proto

```diff
@@ -0,0 +1,107 @@
+syntax = "proto3";
+
+package viam.app.slammapping.v1;
+
+import "app/data/v1/data.proto";
+import "common/v1/common.proto";
+import "google/protobuf/struct.proto";
+import "google/protobuf/timestamp.proto";
+
+option go_package = "go.viam.com/api/app/slammapping/v1";
+
+service CloudSLAMService {
+    rpc StartMappingSession(StartMappingSessionRequest) returns (StartMappingSessionResponse);
+    rpc GetActiveMappingSessionsForRobot(GetActiveMappingSessionsForRobotRequest) returns (GetActiveMappingSessionsForRobotResponse);
+    rpc GetMappingSessionPointCloud(GetMappingSessionPointCloudRequest) returns (GetMappingSessionPointCloudResponse);
+    rpc ListMappingSessions(ListMappingSessionsRequest) returns (ListMappingSessionsResponse);
+    rpc StopMappingSession(StopMappingSessionRequest) returns (StopMappingSessionResponse);
+}
+
+// StartMappingSession
+
+message StartMappingSessionRequest {
+    // the config required to run slam. This includes a replay sensor and map package  
+    google.protobuf.Struct slam_config = 1; //for cartographer/algo specific fields
+    string slam_version = 2; // version to use for slam, defaults stable
+    string map_name = 3;
+}  
+
+message StartMappingSessionResponse {
+    string session_id = 1;
+}
+  
+// GetMappingSession
+
+message GetMappingSessionRequest {
```

Comment:
Do we need a corresponding Response?

Reply:
thanks for catching! this is left over from the poc, not needed anymore (covered by getmappingsessionpointcloudrequest/response)

Reply:
Ah right, thanks!